app/main.py

```diff
@@ -1,13 +1,15 @@
-# from models import Actor
-# from managers import ActorManager
-#
-# if __name__ == "__main__":
-#     Actor.objects = ActorManager()
-#
-#     Actor.objects.create(first_name="Emma", last_name="Watson")
-#     Actor.objects.create(first_name="Daniel", last_name="Radclife")
-#     print(Actor.objects.all())
-#     Actor.objects.update(2, "Daniel", "Radcliffe")
-#     print(Actor.objects.all())
-#     Actor.objects.delete(1)
+from models import Actor
+from managers import ActorManager
+
+if __name__ == "__main__":
+    Actor.objects = ActorManager()
+    print(Actor.objects.all())
+
+
+    # Actor.objects.create(first_name="Emma", last_name="Watson")
+    # Actor.objects.create(first_name="Daniel", last_name="Radclife")
+    # print(Actor.objects.all())
+    # Actor.objects.update(2, "Daniel", "Radcliffe")
 #     print(Actor.objects.all())
+#     Actor.objects.delete(10)
```

Comment:
To fully test the CRUD operations of the `ActorManager`, consider uncommenting these lines. This will allow you to test the `create`, `update`, and `delete` methods effectively.